.github/workflows/npm-package-release.yml

```
@@ -1,12 +1,13 @@
 name: NPM Packages Release
-on:
-  push:
-    branches:
-      - main
+on: [push]
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)